networking/network_policy/multitenant-network-policy.adoc

```diff
@@ -1,12 +1,17 @@
 [id="multitenant-network-policy"]
-= Configuring multitenant mode with network policy
+= Configuring multitenant isolation with network policy
 include::modules/common-attributes.adoc[]
 :context: multitenant-network-policy
 
 toc::[]
 
 As a cluster administrator, you can configure your network policies to provide multitenant network isolation.
 
+[NOTE]
+====
+If you are using the OpenShift SDN cluster network provider, configuring network policies as described in this section provides network isolation similar to `mode: multitenant` but with `mode: networkpolicy` set.
```

Comment:
```suggestion
If you are using the OpenShift SDN cluster network provider, configuring network policies as described in this section provides network isolation similar to `mode: Multitenant` but with `mode: NetworkPolicy` set.
```

(but I don't think that's how we normally talk about the different modes in the docs anyway?)